code/datums/mind/_mind.dm

```diff
@@ -503,16 +503,16 @@
 /// Sets us to the passed job datum, then greets them to their new job.
 /// Use this one for when you're assigning this mind to a new job for the first time,
 /// or for when someone's recieving a job they'd really want to be greeted to.
-/datum/mind/proc/set_assigned_role_with_greeting(datum/job/new_role)
+/datum/mind/proc/set_assigned_role_with_greeting(datum/job/new_role, client/incoming_client)
 	. = set_assigned_role(new_role)
```

Comment:
couldn't we just tack this to a new var and return that at the end of the proc since we don't really need funny dot operator anymore?

Reply:
alrightie then